LetsDinnerV2 MessagesExtension/Controller/RecipeCreationVC/RecipeCreationViewController.swift

```diff
@@ -724,10 +752,27 @@ class RecipeCreationViewController: UIViewController  {
     }
     
     // MARK: Did Tap Buttons
-    @IBAction func didTapDone(_ sender: Any) {
+    @IBAction func didTapDoneButton(_ sender: Any) {
         if editingMode {
-            self.presentDoneActionSheet()
+            
+            if editExistingRecipe {
```

Comment:
That's a lot of if else and difficult to read, no possibility to refactor?

Reply:
updated